src/main/scala/firrtl/Emitter.scala

```diff
@@ -999,7 +999,21 @@ class VerilogEmitter extends SeqTransform with Emitter {
         for (x <- attachAliases) emit(Seq(tab, x))
         emit(Seq("`endif"))
       }
+
+      for ((clk, content) <- noResetAlwaysBlocks if content.nonEmpty) {
+        emit(Seq(tab, "always @(posedge ", clk, ") begin"))
+        for (line <- content) emit(Seq(tab, tab, line))
+        emit(Seq(tab, "end"))
+      }
+
+      for ((clk, reset, content) <- asyncResetAlwaysBlocks if content.nonEmpty) {
+        emit(Seq(tab, "always @(posedge ", clk, " or posedge ", reset, ") begin"))
+        for (line <- content) emit(Seq(tab, tab, line))
+        emit(Seq(tab, "end"))
+      }
+
       if (initials.nonEmpty || ifdefInitials.nonEmpty) {
+        emit(Seq("// Register and memory initialization"))
```

Comment:
This isn't showing up in the example Verilog you shared. In any case I think it's unnecessary, I want to reduce the number of lines, not increase it 🙂

Reply:
Sorry, it actually is generated; I must've generated that Verilog before I added the comment. Should I take it out? I added it on @azidar's suggestion.

Reply:
I think Schuyler put it well https://github.com/freechipsproject/firrtl/pull/1613#pullrequestreview-412129349. Anyway this comment is fine as merged and we can improve it in follow on PRs